include/SW_Weather.h

```diff
@@ -46,6 +46,16 @@
 extern "C" {
 #endif
 
+/* Weather generation methods */
+
+/** No weather generation, only historical data */
+static const unsigned int wgHist = 0;
+
+/** Weather generation method LOCF, see generateMissingWeather() */
+static const unsigned int wgLOCF = 1;
+
+/** Markov weather generator method, see generateMissingWeather() */
+static const unsigned int wgMKV = 2;

```

Comment:
- SW_MKV_setup() has two instances with `Weather_genWeathMethod == 2` that should now be `Weather_genWeathMethod == wgMKV`
- generateMissingWeather() has `method == 0` that should now be `method == wgHist`; it also has `method > 2` that should probably be `method != wgLOCF && method != wgMKV`